.github/workflows/ci.yml

```diff
@@ -24,7 +24,7 @@ jobs:
           php-version: ${{ matrix.php }}
 
       - name: Clone simpletest
-        run: git clone --depth=50 https://github.com/ezyang/simpletest.git
+        run: git clone --depth=50 -b php8-fixes https://github.com/bytestream/simpletest.git
```

Comment:
I sent a PR to https://github.com/ezyang/simpletest/pull/1 if you want to continue using your repo.